Before: 4ff4c88cd3fc249ddbe066a7392af5b2764f07149b87aa8bb2a96bb31191f92b
After: 62489c438f659f4175dbbb3ee4bd753239e683ac0585c155af01d1a99ae97247
Patch: ba7c3aa4ab3b8c357fe2fb0ebb490040bb2ff0dc4c9a7993e1710131eb9df501
--- test/bootstrap/all.rb	2012-07-22 20:44:49.302250549 +1000
+++ test/bootstrap/all.rb	2012-07-22 21:28:18.650089528 +1000
@@ -2,5 +2,3 @@
 APP_ROOT = File.expand_path('../../..',__FILE__)
 require 'rubygems'
 require 'corvid/test/bootstrap/all'
-
-Bundler.require :default
